kirby/api/log.py

```diff
@@ -0,0 +1,24 @@
+from .queue import Queue
+
+LOGGER_TOPIC_NAME = "_logs"
+
+
+class Logger:
+    def __init__(self):
+        self.queue = Queue(LOGGER_TOPIC_NAME)
+        self.name = __name__
+
+    def _send_log_factory(self, level):
+        if level == "log":
```

Comment:
This could use some comments about why you overwrite a very specific value in this function.

Reply:
Comments added